libindy/src/api/payments.rs

```diff
@@ -544,12 +542,15 @@ pub extern fn indy_add_request_fees(command_handle: i32,
 /// resp_json: response for Indy request with fees
 ///   Note: this param will be used to determine payment_method
 ///
+/// Note this endpoint is EXPERIMENTAL. Function signature and behaviour may change
+/// in the future releases.
+///
 /// #Returns
-/// utxo_json - parsed (payment method and node version agnostic) utxo info as json:
+/// receipts_json - parsed (payment method and node version agnostic) receipts info as json:
 ///   [{
-///      txo: <str>, // UTXO input
-///      paymentAddress: <str>, //payment address for this UTXO
-///      amount: <int>, // amount of tokens in this input
+///      receipt: <str>, // receipt
```

Comment:
receipt: <str>, // receipt that can be used for payment referencing and verification